actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -268,6 +262,8 @@ def image_tag(source, options = {})
         tag("img", options)
       end
 
+      # This method will be deprecated in future versions of Rails.
```

Comment:
Though we let the deprecation warning itself suffice, so need for a comment here.